source/clear-linux/guides/deploy-at-scale.rst

```diff
@@ -0,0 +1,342 @@
+.. _deploy-at-scale.rst:
+
+Deploying |CL| at Scale
+#######################
+
+Once you are comfrotable with `Clear Linux concepts`_, your next step
+as a system administraotr may be needing to understand how to deploy 
+|CL| at scale in your enoviornment.  
+
+Throughout this document the term *endpoint* will be used to generally refer 
+to a system targeted for |CL| installation, whether that is a datacenter 
+system or unit deployed in field.  
+
+.. note::
+
+    This is not a replacement or blueprint for designing your IT operating 
+    environment. These are simply recommendations that should be  implemented 
+    with consideration. 
+    
+    Your |CL| deployment should complement the 
+    existing environment and available tools. It is assumed foundational core IT 
+    dependencies, such as network, in your environment are in a healthy and scaled
+    to suit the deployment.
+
+
+
+.. contents:: :local:
+    :depth: 2
+
+
+
+
+
+Pick a |CL| usage and update strategy
+=====================================
+
+Different business scenarios can call for different deployment methodologies. 
+|CLOSIA| offers the flexibility to continue consuming upstream |CL|
+distribution or the option to fork away from the |CL| distribution and
+act as your own OSV. 
+
+
+Below are overviews of both approaches and some considerations.
+ 
+
+
+
+Option #1: Use the |CL| as the upstream origin (mixin)
+------------------------------------------------------
+
+This approach is the *easier to adopt* by relying on the |CL| upstream for 
```

Comment:
either remove "the" or define what "easier to adopt" is.  method?  process?

Reply:
Hm, I understand how this could be confusing. 

The "approach" that is being indicated is the header directly above it. Perhaps "option" is better than "approach" so it's clear that we're talking about the same header or maybe just explicitly saying "This option of using |CL| as the upsteam origin"...